### PR TITLE
CTED-2430 | Tag Forum Notification Link Unclickable Issue

### PR DESCRIPTION
**Pull request by BrianNyatsine**

### CODE REVIEW

**Review by brett-fritz — COMMENTED** [review]
Should it be http? Live clients are on on https.

**Comment by BrianNyatsine** [comment]
> Should it be http? Live clients are on on https.

Generally prod environments running on https should have ssl redirects enabled and for non prod (Non ssl) environments the link will still work

**Review by brett-fritz — APPROVED** [review]

File: classes/observer.php [hunk 1]
```diff
@@ -186,7 +186,7 @@ public static function email_mention_hsu(\mod_hsuforum\event\assessable_uploaded
                         $body = get_config('local_mention_users', 'defaultproperties_body');
                         $body = str_replace("{student_first_name}", $taggedusername, $body);
                         $body = str_replace("{coach_first_name}", $course_coach->firstname, $body);
-                        $body = str_replace("{post_link}", $link, $body);
+                        $body = str_replace("{post_link}", 'http://' .$link, $body);
```
Comment (BrendonPretorius) [inline classes/observer.php:189]: Super minor nit but missing a space between . and $link

File: classes/observer.php [hunk 2]
```diff
@@ -236,7 +236,7 @@ public static function email_mention_hsu(\mod_hsuforum\event\assessable_uploaded
                     $body = get_config('local_mention_users', 'defaultproperties_body');
                     $body = str_replace("{student_first_name}", $taggedusername, $body);
                     $body = str_replace("{coach_first_name}", $course_coach->firstname, $body);
-                    $body = str_replace("{post_link}", $link, $body);
+                    $body = str_replace("{post_link}", 'http://' .$link, $body);
```
Comment (BrendonPretorius) [inline classes/observer.php:239]: Same as above

**Review by BrendonPretorius — CHANGES_REQUESTED** [review]

File: classes/observer.php (hunk 1 above)
Comment (BrianNyatsine) [reply classes/observer.php]: thanks, fixed this

File: classes/observer.php (hunk 2 above)
Comment (BrianNyatsine) [reply classes/observer.php:239]: done

Comment (BrianNyatsine) [reply classes/observer.php:239]: done

**Review by BrendonPretorius — APPROVED** [review]

**Pull request merged**